get_logs.py

```diff
@@ -33,36 +36,14 @@ def get_logs(ec2, ec2_client, cw_client, args):
     # Fetch running instances from PicSiteASG autoscaling group
     instances = ec2.instances.filter(
             Filters=[
-                {'Name': 'instance-state-name', 'Values': ['running']},
                 {'Name': 'tag:aws:autoscaling:groupName', 'Values': ['PicSiteASG']}
             ]
     )
+    instance_id_list = [instance.id for instance in instances]
 
-    # Get ids from retrieved instances but exclude if not 'ok' status
```

Comment:
Good call here, some instances may have shutdown (if our tests ever took it eassyyy!)